code/__DEFINES/flags.dm

```diff
@@ -24,6 +24,7 @@
 #define NOSLIP		1024 		//prevents from slipping on wet floors, in space etc (NOTE: flag shared with THICKMATERIAL for external suits and helmet)
 
 #define OPENCONTAINER	4096	// is an open container for chemistry purposes
+#define TRANSPARENT	8192

```

Comment:
@optimumtact I did test though if having a syringe/dropper would either make you immune from injections like hardsuits do, or if it would block gas smoke effects. Neither appears to have occurred.